modules/PhyVersoBSP/connector_1/evse_board_supportImpl.cpp

```diff
@@ -59,15 +59,6 @@ void evse_board_supportImpl::init() {
 void evse_board_supportImpl::ready() {
 }
 
-void evse_board_supportImpl::handle_setup(bool& three_phases, bool& has_ventilation, std::string& country_code) {
```

Comment:
Looks like you need to publish the caps for connector 1 as well like its done for connector 2